pyproject.toml

```
@@ -8,7 +8,7 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "ops-scenario"
 
-version = "5.4"
+version = "5.5"
```

```suggestion
version = "5.4.1"
```

a patch bump is enough for a backwards-compatible bugfix